srml/balances/src/lib.rs

```diff
@@ -14,13 +14,149 @@
+//! # Balances Module
+//!
+//! The balances module provides functionality for handling accounts and balances. To use the balances module, you need
+//! to implement the [balances Trait](https://crates.parity.io/srml_balances/trait.Trait.html). Supported dispatchables
+//! are documented in the [`Call` enum](https://crates.parity.io/srml_balances/enum.Call.html).
+//!
+//! ## Overview
+//!
+//! The balances module provides functions for:
+//!
+//! - Getting and setting free balances
+//! - Retrieving total, reserved, and unreserved balances
+//! - Repatriating a reserved balance to a beneficiary account that exists
+//! - Transferring a balance between accounts (when not reserved)
+//! - Slashing an account balance
+//! - Account creation and removal
+//! - Lookup of an index to reclaim an account
+//! - Managing total issuance
+//! - Setting and managing locks
+//!
+//! ### Terminology
+//!
+//! - **Existential Deposit:** The minimum balance required to create or keep an account open. This prevents
+//! "dust accounts" from filling storage.
+//! - **Total Issuance:** The total amount of units in existence in a system.
+//! - **Reaping an account:** The act of removing an account by resetting its nonce. Happens after its balance is set
+//! to zero.
+//! - **Free Balance:** The liquid or spendable balance. The free balance is the only balance that matters for most
+//! operations. When this balance falls below the existential deposit, the account is removed.
+//! - **Reserved Balance:** Reserved balance still belongs to the account holder, but is suspended. Reserved balance
+//! can still be slashed, but only after all of free balance has been slashed. If the reserved balance falls below the
+//! existential deposit then it will be deleted.
+//! - **Imbalance:** A condition when some funds were created or deducted without equal and opposite accounting. Functions
+//! that result in an imbalance will return an object of the `Imbalance` trait that must be handled.
+//! - **Locks:** A freeze on an account's balance until a specified block number.
+//!
+//! ### Implementations
+//!
+//! The balances module provides implementations for the following traits. If these traits provide the functionality
+//! that you need, then you can avoid coupling with the balances module.
+//!
+//! - [`Currency`](https://crates.parity.io/srml_support/traits/trait.Currency.html): Functions for dealing with a
+//! fungible assets system.
+//! - [`LockableCurrency`](https://crates.parity.io/srml_support/traits/trait.LockableCurrency.html): Functions for
+//! dealing with accounts that allow liquidity restrictions.
+//! - [`Imbalance`]: Functions for handling imbalances between total issuance in the system and account balances.
+//! Must be used when a function creates new funds (e.g. a reward) or destroys some funds (e.g. a system fee).
+//! - [`MakePayent`]: Simple trait designed for hooking into a transaction payment. It operates over a single generic
```

Comment:
I would remove too specific trait doc,
```suggestion
//! - [`MakePayent`]: Simple trait designed for hooking into a transaction payment.
```